third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -99,6 +99,28 @@ std::string getRegisterSizeCode(int size, bool is_float) {
   }
 }
 
+Value createL2CachePolicy(triton::CacheModifier l2Cache,
+                        ConversionPatternRewriter &rewriter, Location loc) {
+  // Only create a policy register if we need L2 cache control
+  if (l2Cache == triton::CacheModifier::NONE)
+    return Value();
+    
+  PTXBuilder ptxBuilder;
+  auto &policy = ptxBuilder.create<>("createpolicy.fractional")
+      ->o("L2::evict_first", l2Cache == triton::CacheModifier::EVICT_FIRST)
```

Comment:
where is this defined CacheModifier::EVICT_FIRST?

Reply:
Similar as above. Fixed to correctly be EvictionPolicy::EVICT_FIRST